Source/WTF/wtf/cocoa/SpanCocoa.h

```diff
@@ -41,6 +41,12 @@ inline Span<const std::byte> asBytes(const RetainPtr<NSData>& data)
     return asBytes(data.get());
 }
 
+inline Span<const uint8_t> spanFromNSData(NSData* data)
```

Comment:
Any change we could use asBytes() from SpanCocoa.h?

Reply:
I tried this, converting everything to use std::byte instead of uint8_t turns out to be pretty involved as Span<const uint8_t> and Span<const std::byte> don't seem to be easily interchangeable.

Reply:
Ok, maybe we can name it `asUInt8Span()` to be a little more consistent with `asBytes()`.